README.md

```diff
@@ -1,24 +1,76 @@
-# README
+# RAILS TEMPLATE 
+This repo is a template to be used for creating new projects.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## System dependencies                                                                                                                                  
+  - Ruby (see .ruby-version)
+  - Rails 7.0+
+  - Hotwire (Stimulus+Turbo)
+  - MySQL 8+  
+  - Redis 6.2+
 
-Things you may want to cover:
+## Template Notes
+After creating your new project, you will need to touch some files to customize the new environment. Those files are:  
 
-* Ruby version
+- config/deploy/* to provide the host names for each deployment target.
+- config/deploy.rb to update the project name & other deployment settings.
+- Look at `config/initializers/site_information.rb` and make changes as needed to the constant values shown there.
+- Perform a 'find all' to look for 'fmaprivacy' and 'fmadata'. Modify those values as needed.
+- You will also need to go through the views to remove/change some of the current boiler plate, logo, etc.
+- This template is configured for the free version of Sidekiq. Review the Gemfile and config/routes.rb files if you are using Sidekiq Pro.
+- Change the name of the application in `config/application.rb`. Change FMAdataTemplate to something meaningful.
 
-* System dependencies
+NOTE: If you find other files that need to be touched after creating a new project please come back here and update the documentation!
 
-* Configuration
+## Configuration and installation
 
-* Database creation
+### Tailwind
 
-* Database initialization
+In order to use Tailwind make sure to run `bundle exec rails assets:precompile`.
+For development environment use `./bin/dev` instead of `rails s` for running the app. It runs both puma and tailwind
+that detects changes to css. 
 
-* How to run the test suite
+### Credentials 
 
-* Services (job queues, cache servers, search engines, etc.)
+See Keybase for master.key for this project.
+To generate credentials, run:
+  - in vim use `EDITOR=editor_of_your_choice rails credentials:edit`
+Enter your keys, save and exit. See config/credentials.example.yml for examples.
 
-* Deployment instructions
+Set up locking strategy during configuration (https://github.com/plataformatec/devise/wiki/How-To:-Add-:lockable-to-Users).
 
-* ...
+### Gems
+
+Run `bundle install` command to install Rails and all other Gem dependencies. 
+
+### Database creation
+
+MySQL 8 is the recommended version to run on your machine. Once you have MySQL running, run:
+  `cp config/database.example.yml config/database.yml`
+and configure your config/database.yml to your machine's MySQL configuration. Minor edits should only be necessary.
+
+Run rake db:setup to create and migrate the database from schema (db/schema.rb).
```

Comment:
This part will be done by bin/setup script, so you can remove that.
I'd only leave creating database.yml to the users, as everyone might have different users and passwords configured on their machines.